autopilot/autopilot_california_housing.ipynb

```diff
@@ -0,0 +1,900 @@
+{
```

Comment:
<p>The above cell mentions "the following code, when uncommented, will delete the local files used in this demo" - but the cell is not commented. I do recommend commenting it out, to reduce the chance that a user accidentally deletes files and directories that they didn't intend to. It's most important to delete resources that will incur cost over time.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>